.github/workflows/build.yaml

```diff
@@ -2,9 +2,7 @@ name: app
 on:
   push:
     branches:
-      - '*'
-    tags:
-      - v*.*.*
+      - '**'
```

Comment:
I noticed the workflow wasn't running on the dependabot branches. Dependabot creates branches with a `/` in the name, and the wildcard only matches up to the first path segment. If we were to turn off Dependabot and use Renovate, it also makes branches with a similar pattern. 

https://github.com/orgs/community/discussions/25445